include/SDL3/SDL_events.h

```diff
@@ -154,6 +154,7 @@ typedef enum SDL_EventType
     SDL_EVENT_MOUSE_WHEEL,             /**< Mouse wheel motion */
     SDL_EVENT_MOUSE_ADDED,             /**< A new mouse has been inserted into the system */
     SDL_EVENT_MOUSE_REMOVED,           /**< A mouse has been removed */
+    SDL_EVENT_MOUSE_RAWMOVE,           /**< Mouse moved (raw devie counts) */
```

Comment:
I'm not entirely sure how this concept would fit in with regular mouse motion, but I would probably name this something like `SDL_EVENT_MOUSE_RELATIVE_MOTION`

Reply:
Or maybe `SDL_EVENT_MOUSE_RAW_MOTION`